features/storage/TESTS/filesystem/general_filesystem/main.cpp

```diff
@@ -1027,9 +1027,10 @@ static void FS_fseek_beyond_empty_file_seek_set()
 
     int read_sz = fread(read_buf, sizeof(char), sizeof(read_buf), fd[0]);
     TEST_ASSERT_EQUAL(0, read_sz);
-
+#if !defined(__MICROLIB)
```

Comment:
On this file it is only the assertion that should be compiled out as the call to feof() is valid even for Microlib. Also I would probably add `// feof() always returns 0 because the EOF indicator is not supported by Microlib` as a comment.

Reply:
All the test cases are not combined filehandle API(fseek,fclose etc) with the TEST_ASSERT*. so I don't want to modify but I can add the comment.